app/src/main/java/com/belatrixsf/allstars/ui/contacts/ContactFragment.java

```diff
@@ -158,10 +172,10 @@ public void beforeTextChanged(CharSequence s, int start, int count, int after) {
                 @Override
                 public void onTextChanged(CharSequence s, int start, int before, int count) {
                     if (s.length()>0){
```

Comment:
This validation should go to the presenter. all your business logic in the presenter. then the view receives the answer if it should show or not the view(s)